src/test/kotlin/uk/gov/dluhc/printapi/mapper/PrintRequestMapperTest.kt

```diff
@@ -46,51 +49,32 @@ class PrintRequestMapperTest {
     @Mock
     private lateinit var instantMapper: InstantMapper
 
-    @BeforeEach
-    fun setup() {
-        mapper = PrintRequestMapperImpl()
-        ReflectionTestUtils.setField(mapper, "idFactory", idFactory)
-        ReflectionTestUtils.setField(mapper, "clock", FIXED_CLOCK)
-        ReflectionTestUtils.setField(mapper, "instantMapper", instantMapper)
-    }
+    @Mock
+    private lateinit var supportingInformationFormatMapper: SupportingInformationFormatMapper
+
+    @Spy
+    private val clock: Clock = FIXED_CLOCK
 
     @ParameterizedTest
     @CsvSource(value = ["EN, EN", "CY, CY"])
-    fun `should map send application to print message to print details`(
+    fun `should map send application to print message to print request`(
         certificateLanguageModel: CertificateLanguageModel,
         certificateLanguageEntity: CertificateLanguageEntity
     ) {
         // Given
         val ero = buildEroDto()
-        val message = buildSendApplicationToPrintMessage(certificateLanguage = certificateLanguageModel)
+        val supportingInformationFormatModelEnum = EASY_MINUS_READ
+        val supportingInformationFormatEntityEnum = EASY_READ
+        val message = buildSendApplicationToPrintMessage(
+            certificateLanguage = certificateLanguageModel,
+            supportingInformationFormat = supportingInformationFormatModelEnum
+        )
         val requestId = aValidRequestId()
         given(idFactory.requestId()).willReturn(requestId)
-        val expectedEnglishEroContactDetails = ElectoralRegistrationOffice(
-            name = "Gwynedd Council Elections",
-            phoneNumber = "01766 771000",
-            website = "https://www.gwynedd.llyw.cymru/en/Council/Contact-us/Contact-us.aspx",
-            emailAddress = "TrethCyngor@gwynedd.llyw.cymru",
-            address = Address(
-                property = "Gwynedd Council Headquarters",
-                street = "Shirehall Street",
-                town = "Caernarfon",
-                area = "Gwynedd",
-                postcode = "LL55 1SH",
-            )
-        )
-        val expectedWelshEroContactDetails = ElectoralRegistrationOffice(
-            name = "Etholiadau Cyngor Gwynedd",
-            phoneNumber = "01766 771000",
-            website = "https://www.gwynedd.llyw.cymru/cy/Cyngor/Cysylltu-%c3%a2-ni/Cysylltu-%c3%a2-ni.aspx",
-            emailAddress = "TrethCyngor@gwynedd.llyw.cymru",
-            address = Address(
-                property = "Pencadlys Cyngor Gwynedd",
-                street = "Stryd y Jêl",
-                town = "Caernarfon",
-                area = "Gwynedd",
-                postcode = "LL55 1SH",
-            )
-        )
+        given(supportingInformationFormatMapper.toPrintRequestEntityEnum(any()))
+            .willReturn(supportingInformationFormatEntityEnum)
+        val expectedEnglishEroContactDetails = toEroContactDetails(ero.englishContactDetails)
```

Comment:
I changed these, so it is more explicit that the expected values are derived from the `eroDto` and is easier to read and understand.

Reply:
Thanks, @Neil-Massey , for your suggestion.
Added it as an extension method for tests.

<pre>
val expectedEnglishEroContactDetails = ero.englishContactDetails.toElectoralRegistrationOffice()
val expectedWelshEroContactDetails = ero.welshContactDetails!!.toElectoralRegistrationOffice()
</pre>

Reply:
<pre>
package uk.gov.dluhc.printapi.testsupport.testdata.dto

fun EroContactDetailsDto.toElectoralRegistrationOffice() =
    ElectoralRegistrationOffice(
        name = name,
        phoneNumber = phoneNumber,
        website = website,
        emailAddress = emailAddress,
        address = with(address) {
            Address(
                property = property,
                street = street,
                town = town,
                area = area,
                postcode = postcode,
            )
        }
    )
</pre>